pkg/apis/enmasse/v1/types_messaginguser.go

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+/*
+ * Be careful with the comments in this file. The prefix "+" indicates that this is being processed
+ * by the client generator. The location, empty lines, and other comments in this file may confuse
+ * the generator, and produce a non-version version.
+ */
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:shortName=msgusr;msguser;msgusers,categories=enmasse
```

Comment:
please add also `msgu`, as we have for rest resources like msgt, msgp, msga

Reply:
done.

Reply:
Thank you!